build.sbt

```diff
@@ -39,8 +37,7 @@ lazy val buildSettings = Seq(
         <email>dev-role@allenai.org</email>
       </developer>
     </developers>,
```

Comment:
I think so. This is what I followed: https://github.com/allenai/common/blob/master/build.sbt .
That stuff shows up under org info in Bintray.